server/main.go

```diff
@@ -73,6 +76,28 @@ func main() {
 	prometheus.MustRegister(exporter, metricExporter)
 	mux.Handle("/metrics", promhttp.Handler())
 
-	err := http.ListenAndServe(cli.HTTPAddress, mux)
-	ctx.FatalIfErrorf(err)
+	go func() {
+		err := http.ListenAndServe(cli.HTTPAddress, mux)
+		ctx.FatalIfErrorf(err)
+	}()
+
+	workerClient := worker.Client{
+		GrafanaUrl:   cli.GrafanaUrl,
+		Token:        cli.DashboardUpdateToken,
+		AnalyticsUrl: cli.HTTPAddress,
+		Logger:       logger,
+	}
+
+	// Run once per 24 hours
+	ticker := time.NewTicker(24 * time.Hour)
```

Comment:
Could make this timeout configurable, also http endpoint to trigger run would be nice addition, it would make it easier to test.